# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qemu/no-xen-emu.patch.x32
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- qemu-9.1.1/hw/i386/Kconfig.vanilla	2024-11-16 23:15:43.170023637 +0100
+++ qemu-9.1.1/hw/i386/Kconfig	2024-11-16 23:15:51.223357117 +0100
@@ -150,5 +150,5 @@
 
 config XEN_EMU
     bool
-    default y
+    default n
     depends on KVM && I386
